grakn-graph/src/test/java/ai/grakn/graph/property/RelationPropertyTest.java

```diff
@@ -48,41 +44,6 @@
     public final ExpectedException exception = ExpectedException.none();
 
     @Property
-    public void whenAddingARolePlayer_ItIsAddedToTheCollectionOfRolePlayers(
-            Relation relation, @FromGraph RoleType roleType, @FromGraph Instance rolePlayer) {
-
-        assumeFalse(rolePlayer.isResource() && rolePlayer.asResource().type().isUnique());
-
-        relation.addRolePlayer(roleType, rolePlayer);
-
-        assertThat(relation.rolePlayers(), hasItem(rolePlayer));
-    }
-
-    @Property
-    public void whenAddingARolePlayerPlayingARole_TheRolePlayerIsAddedToTheCollectionOfRolePlayersForThatRole(
-            Relation relation, @FromGraph RoleType roleType, @FromGraph Instance rolePlayer) {
-
-        assumeFalse(rolePlayer.isResource() && rolePlayer.asResource().type().isUnique());
-
-        relation.addRolePlayer(roleType, rolePlayer);
-
-        assertThat(relation.rolePlayers(roleType), hasItem(rolePlayer));
-    }
-
-    @Property
-    public void whenAddingARolePlayer_NoRolePlayersAreRemoved(
-            Relation relation, @FromGraph RoleType roleType, @FromGraph Instance rolePlayer) {
-
-        assumeFalse(rolePlayer.isResource() && rolePlayer.asResource().type().isUnique());
-
-        Instance[] rolePlayers = relation.rolePlayers(roleType).toArray(new Instance[0]);
-
-        relation.addRolePlayer(roleType, rolePlayer);
-
-        assertThat(relation.rolePlayers(roleType), hasItems(rolePlayers));
-    }
-
-    @Property
```

Comment:
These tests are for multiple role-players, they don't need to be deleted. They only checked the resource was not unique so they could be sure they could put that resource in new relationships.